engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -1528,6 +1528,35 @@ - (FlutterDartProject*)project {
   return self.dartProject;
 }
 
+- (void)sendDeepLinkToFramework:(NSURL*)url completionHandler:(void (^)(BOOL success))completion {
+  __weak FlutterEngine* weakSelf = self;
+  [self waitForFirstFrame:3.0
```

Comment:
We need this due to the existing behavior that iOS deeplink will first launch the app and then a random delay later it will sends a [UIApplication openUrl]. that is why we need a delay to wait for first frame. Is this still the case for UIScene?

Reply:
Yes, we still need the delay